src/utils/validateName.ts

```diff
@@ -35,6 +35,12 @@ export function validateMnemonics(mnemonics: string): string | void {
     return "Mnemonics are can only be composed of a non-alphanumeric character or a whitespace.";
 }
 
+export const  validateThreeBotName = (name: string): string | void => {
```

Comment:
The names validations are already done with the method `validateName` and it looks like this one doesn't add extra validations. can we use `validateName` instead? 
And as in the `validateName` it should not exceeds 15 char if is it will break the deployment